e2e/testdata/fn-render/path-index-duplicate/.expected/config.yaml

```diff
@@ -15,3 +15,4 @@
 # One of the functions in the pipeline fails resulting in
 # non-zero exit code and no changes in the resources.
 exitCode: 1
+skip: true
```

Comment:
@droot @natasha41575 `kpt fn render` command will exit with 0 for this case.

Reply:
Didn't understand. If `skip:true`, the test is not even invoked ...right, look for `t.Skip()` ? so not sure why would even be the command be executed.

Reply:
This test can pass after the fix about the exit code. Remove the skip.